src/main/java/client/mvc/View/ViewInformationField.java

```diff
@@ -26,8 +26,12 @@ private void initComponents() {
     @Override
     public void refresh() {
         Cell point = m.getPoint();
-        if (point == null || point.x != -2)
+        if (point == null || !(point.x == -2 || point.x == 100))
             return;
+        if (point.x == 100) {
+            setText("<html>Information about players: <br> You are 1 player <br> Second player are left <br> Waiting a new opponent <br>");
```

Comment:
Second player IS left

Reply:
I tested different situations. Now players are able to leave from game and remaining player will wait new opponent. It seems to work)